src/pages/CatEdit.js

```diff
@@ -1,9 +1,86 @@
 import React from 'react'
+import { useState } from 'react'
+import { Form, FormGroup, Label, Input, Button, Col } from 'reactstrap'
+import { useNavigate, useParams } from 'react-router-dom'
+
+const CatEdit = ({ cats, updateCat }) => {
+
+    const { id } = useParams()
+    let currentCat = cats?.find((cat) => cat.id === +id)
+
+    const [editCat, setEditCat] = useState({
+        name: currentCat?.name,
+        age: currentCat?.age,
+        enjoys: currentCat?.enjoys,
+        image: currentCat?.image
+    })
+    const handleChange = (e) => {
+        setEditCat({ ...editCat, [e.target.name]: e.target.value })
+    }
+
+    const navigate = useNavigate()
+
+    const handleSubmit = () => {
+        updateCat(editCat, currentCat.id)
+        navigate(`/catshow/${currentCat.id}`)
+      }
 
-const CatEdit = () => {
     return (
         <>
-            <h2>CatEdit</h2>
+            <div className="CatNew">
+                <div className="CatNew1">
+                    <h2>Edit Profile</h2>
+                </div>
+                <div className="CatNew2">
+                    <Form>
+                        <FormGroup>
+                            <Label for="name">
+                                Name
+                            </Label>
+                            <Input
+                                name="name"
+                                placeholder="What is your name?"
```

Comment:
I would suggest that instead of using a placeholder, setting the value to what is currently in state for that input.  Hint: Try using the value attribute for input.